_data/simple-icons.json

```diff
@@ -10226,17 +10232,17 @@
         {
             "title": "X.Org",
             "hex": "F28834",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/9/90/X.Org_Logo.svg"
+            "source": "https://en.wikipedia.org/wiki/File:X.Org_Logo.svg"
```

Comment:
Could also be the wikimedia link: https://commons.wikimedia.org/wiki/File:X.Org_Logo.svg